src/hotspot/share/gc/shenandoah/shenandoahSimpleBitMap.inline.hpp

```diff
@@ -27,15 +27,22 @@
 
 #include "gc/shenandoah/shenandoahSimpleBitMap.hpp"
 
+inline uintx tail_mask(uintx bit_number) {
+  if (bit_number >= BitsPerWord) {
+    return -1;
+  }
```

Comment:
When would we call here with `bit_number >= BitsPerWord` ? If never, may be we assert that?

Reply:
It's called with `bit_number == BitsPerWord` (I tried an assertion first).